config.defaults.yaml

```diff
@@ -51,3 +51,7 @@ docs:
       coordinates: [254.7213, 32.8341]
       period: 0.31087445597147706
```

Comment:
Do you want to take a stab at the exceedingly-precise periods for other types of objects?

Reply:
I am less clear what you mean here ;)

Reply:
I bet the answer is no though!

Reply:
@AshishMahabal brought it up elsewhere. I guess it's fine as is.